packages/gatsby-theme/src/blocks/BlogHeadBlock/BlogHeadBlock.stories.mdx

```diff
@@ -7,7 +7,7 @@ export const data = {
   author: {
     name: "Marco José Sánchez",
     link: "/",
-    ariaLabel: "Marco",
+    authorLabel: "Marco",
```

Comment:
thanks for the refactor 🙏  looks good. What do you think if we describe the field as `label` instead of `authorLabel`? I think because it's inside the `author` object would make sense

Reply:
Sure, thank you @edvicaty 🙏